op-deployer/pkg/deployer/bootstrap/flags.go

```diff
@@ -123,6 +123,11 @@ var (
 		Usage:   "Upgrade controller.",
 		EnvVars: deployer.PrefixEnvVar("UPGRADE_CONTROLLER"),
 	}
+	SuperchainProxyAdminFlag = &cli.StringFlag{
+		Name:    "superchain-proxy-admin",
+		Usage:   "Superchain proxy admin.",
```

Comment:
Isn't this an implementation contract? The name makes it seem like a role so would be good to explain in more detail in the `Usage` section here